addons/medical_gui/functions/fnc_onCategorySelected.sqf

```diff
@@ -0,0 +1,25 @@
+#include "script_component.hpp"
+/*
+ * Author: KJW
+ * Updates selected category and page variables on new category selection Used in GUI statements.
+ *
+ * Arguments:
+ * 0: Category <STRING>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * ["triage"] call ace_medical_gui_fnc_onCategorySelected
+ *
+ * Public: No
+ */
+
+ params ["_category"];
+
+if (GVAR(selectedCategory) isEqualTo 'triage') then {
+    GVAR(selectedCategory) = _category;
```

Comment:
minor style thing but move identical code out of the if else blocks

Reply:
Done
`isEqualTo 'triage'` was also meant to be changed to `isEqualTo _category`, forgot to do but resolved in same commit